light-zk.js/src/relayer.ts

```diff
@@ -107,7 +107,7 @@ export class Relayer {
     provider: Provider,
   ): Promise<RelayerSendTransactionsResponse> {
     try {
-      const response = await axios.post(this.url + "/relayInstruction", {
+      const response = await axios.post(this.url + "/relayTransaction", {
```

Comment:
arent we technically still sending instruction**s** ?

Reply:
Yes we are sending several instructions which in concert execute one shielded transaction.
Every instruction is executed in a single solana transaction.

happy for suggestions on better naming!

Reply:
probably `relayInstructions` makes most sense then? but this is not urgent so can still merge this pr now